metadata-ingestion/docs/sources/db2/db2_pre.md

```diff
@@ -0,0 +1,32 @@
+### Permissions
+
+In order to execute this source the user needs `SELECT` privileges on the system schema and tables:
+
+- Db2 for LUW: `SYSCAT.*`
+- Db2 for z/OS: `SYSIBM.*`
+- Db2 for IBM i (AS/400): `QSYS2.*`
+
+Additionally, when profiling is enabled, the user will need `SELECT` privileges on the tables to be profiled.
```

Comment:
Does that lineage require some additional permission?

Reply:
Which lineage?

Reply:
SELECT privilege on the system metadata tables (SYSCAT, SYSIBM, or QSYS2) is required for metadata extraction. I’m wondering if this also covers lineage, or if we need access to additional system tables. Just double-checking, since other sources sometimes require extra tables or schemas.

Reply:
No additional privileges needed, it's all in those system metadata tables.